commune_app/all_models/users.py

```diff
@@ -37,11 +37,11 @@ def save(self, *args, **kwargs):
         self.full_clean()
         return super().save(*args, **kwargs)
 
-    def vote_on_decision(decision_id):
-        pass
-
-    def add_decision(args):
-        pass
-
-    def execute_chore():
-        pass
+    def join_commune(self, commune_id):
+        if (self.commune_id is not None and self.commune_id != commune_id):
+            raise Exception("already partner in another commune")
+        self.commune_id = commune_id
+
+    def leave_commune(self):
+        if (self.commune_id is not None):
+            self.commune_id = None
```

Comment:
maybe you can add here in case the commune_id is None raise an exception 
```suggestion
        if (self.commune_id is None):
            raise Exception("The user is NOT partner in any commune")
        else
            self.commune_id = None
```